src/itest/resources/features/psc_statements.feature

```diff
@@ -89,4 +89,28 @@ Feature: Process Psc Statement Requests
 
     Examples:
       | companyNumber |
-      | OC421554      |
\ No newline at end of file
+      | OC421554      |
+
+  Scenario Outline: Update an existing Psc Statement successfully
+
+    Given Psc statements data api service is running
+    And a psc statement exists for company number "<companyNumber>" with statement id "<statementId>"
+    When I send a PUT request with payload "<data>" file for company number "<companyNumber>" with statement id "<statementId>"
+    Then I should receive 200 status code
+    And the CHS Kafka API is invoked for company number "<companyNumber>" with statement id "<statementId>"
+
+    Examples:
+      | companyNumber | statementId                 | data                      |
+      | OC421554      | DHTUrJoAuKdXw7zvkreyAm_SoH0 | company_psc_statement_put |
+
+  Scenario Outline: Update and existing Psc Statement with legacy psc_statement_id successfully
+
+    Given Psc statements data api service is running
+    And a psc statement exists with legacy data for company number "<companyNumber>" with statement id "<statementId>"
+    When I send a PUT request with payload "<data>" file for company number "<companyNumber>" with statement id "<statementId>"
+    Then I should receive 200 status code
+    And the CHS Kafka API is invoked for company number "<companyNumber>" with statement id "<statementId>"
+
```

Comment:
There should be a verification step which checks the legacy field `psc_statement_id` has been replaced with the new raw ID field `psc_statement_id_raw`.